Pop-Grupa-A-Frontend/src/app/login/component/LoginForm.css

```diff
@@ -0,0 +1,8 @@
+.login-form {
```

Comment:
[styled-components](https://www.styled-components.com/)
polecam użyć tego,aby uniknąć tworzenia takich plików css.
W tej notacji by to wyglądało tak:

```
const StyledCard= styled(Card)`
  width: 30%;
  margin: 0 auto 0 auto;
  & > div {
  margin-bottom: 10px;
  }
`
```